crates/astria-conductor/src/block_cache.rs

```diff
@@ -95,31 +91,6 @@ impl<T: GetSequencerHeight> BlockCache<T> {
             }),
         }
     }
-
-    /// Reschedules a block at the previous height.
-    ///
-    /// # Errors
-    ///
-    /// Returns an error if:
-    /// + the height of the block is not equal to the previous height recorded in the struct.
-    /// + the height of the block is at level 0.
-    pub(crate) fn reschedule_block(&mut self, block: T) -> Result<(), Error> {
```

Comment:
Removed because this was replaced by a simpler rescheduling mechanism - schedule the blocks to be sent once there is space in the channel.